packages/view3d/src/core/PostProcessing.ts

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2020 NAVER Corp.
+ * egjs projects are licensed under the MIT license
+ */
+
+
+import * as THREE from "three";
+import { EffectComposer } from "three/examples/jsm/postprocessing/EffectComposer";
+import { SSAOPass } from "three/examples/jsm/postprocessing/SSAOPass";
+import { RenderPass } from "three/examples/jsm/postprocessing/RenderPass";
+import { UnrealBloomPass } from "three/examples/jsm/postprocessing/UnrealBloomPass";
+import { PerspectiveCamera, Scene, Vector2, WebGLRenderer } from "three";
```

Comment:
Please do not directly import three.js components, but use them in the form `THREE.XXX`

Reply:
May i ask you reason? I guess for treeshaking. right?

Reply:
Well, it's for clarity. As it shows where the module came from directly.
It won't increase the bundle size but as it's already importing three modules as `THREE` you can just use it.